blocks/serializer/index.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import * as query from 'hpq';
+
+/**
+ * Takes a block list and returns the serialized post content
+ *
+ * @param  {Array}  blocks Block list
+ * @return {String}        The post content
+ */
+export default function serialize( blocks ) {
+	return blocks.reduce( ( memo, block ) => {
```

Comment:
This block is a bit hard to follow. It may be good to separate the attribute handling to separate functions, and possibly add some comments.